.github/workflows/checks.yml

```diff
@@ -0,0 +1,28 @@
+name: Checks
+
+on:
+  workflow_call:
+
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: 'Checkout'
+        uses: actions/checkout@v4
+      - name: 'Run check task'
```

Comment:
Maybe it need to setup java 11? same with the publish of main.

Reply:
done